packages/shared/package.json

```diff
@@ -100,6 +100,7 @@
     "typescript": "^4.4.4"
   },
   "dependencies": {
+    "@tailwindcss/line-clamp": "^0.2.2",
```

Comment:
Cool!

Could you also check the current use in the `Card.module.css` there is a "Static" fix there for line clamps.
Would suggest we uniform it.

Reply:
Awesome! Thank you for mentioning that, I might just use that.